SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -36,7 +36,7 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date',
       .when('/login', {
```

Comment:
@mohameddahrouj  Reverted back but I noticed you use spaces instead of tabs so its not even indented properly (ex line 31 has 6 spaces which should be 8 or 2 tabs).

Reply:
I agree, if only there was a half tab.